### ISSUE TITLE
Configure Elasticsearch Service from external server instead of local[v5]

### ISSUE THREAD

**Issue by vasanth1896** [reportportal/reportportal#1095]
https://github.com/reportportal/reportportal/issues/764#issue-532095841

Please refer the above issue for details. Was able to configure ES_HOST until 5.2.3v, but after upgrading to 5.3v its not working/ Kindly help on this. If i am doing anything wrong in compose file, please let me know.

```
analyzer:
    image: 'service-auto-analyzer:5.3.0'
    environment:
      LOGGING_LEVEL: info
      AMQP_URL: amqp://rabbitmq:rabbitmq@rabbitmq:5672
    environment:
      - ES_HOSTS=<HOST_NAME>
    #depends_on:
      #- rabbitmq
      #- elasticsearch
    restart: always
```

**Comment by Cyberglamdring** [comment]
Hello @vasanth1896,
[Check](https://github.com/reportportal/reportportal/blob/master/docker-compose.yml#L55) please the correct path to the image `reportportal/service-auto-analyzer:5.3.0`

**Comment by vasanth1896** [comment]
Thanks for update @Cyberglamdring . 
Even after updating the path same issue still persist. And Auto analyser service is running fine. Just that its not picking up ES_HOST.
In Report Portal i am getting this error while trying to assign defect type,
'Impossible interact with integration. There are no analyzer services with search logs support deployed.'

**Comment by vasanth1896** [comment]
Hi @Cyberglamdring , Any Workaround for this issue. Kindly assist on this. Help is much appreciated. Thanks.

**Comment by vasanth1896** [comment]
Now it got fixed thanks, after updating to 5.3.1 :)
